handlers.go

```diff
@@ -59,6 +59,10 @@ func getPageHandler(w Response, r Request) Output {
 	page := NewPage(r.PathValue("page"))
 
 	if !page.Exists() {
+		if s, err := os.Stat(page.Name()); err == nil && s.IsDir() {
+			// TODO now it redirects to index (md or html). check if the files exists, otherwise render default template. template should just print all childrent notes
+			return Redirect("index")
```

Comment:
I think this shouldn't be "index" but the constant `INDEX` as the user can change the index file name from `index` to `README` to `home`

Reply:
that makes totally sense!